rep-2000.rst

```diff
@@ -547,6 +547,124 @@ Build System Support:
+| OpenCV      |     3.2.0      |     4.1.0     |     3.4.6*     |      3.2.0     |        3.2.0      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenSSL     |     1.1.1c     |     1.0.2r    |     1.0.2r     |      1.1.1c    |1.1.1b / 1.0.2r****|
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Poco        |     1.8.0      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.0      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Python      |     3.6.5      |     3.7.3     |     3.7.3      |      3.7.3     |        3.7.2      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Qt          |     5.9.5      |     5.12.3    |     5.10.0     |      5.11.3    | 5.11.3 / 5.6.3****|
++-------------+----------------+---------------+----------------+----------------+-------------------+
+|                              |         **Linux only**                                              |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| PCL         |     1.8.1      |     N/A       |     N/A        |      1.9.1     |        1.8.1      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+|                                **RMW DDS Milddleware Providers**                                   |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Connext DDS |                              5.3.1***           |               N/A                  |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Cyclone DDS |                                 0.5.0 (Eusebius)                                     |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+|  Fast-RTPS  |                                      1.9.0                                           |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenSplice  |                                  6.9.190705OSS                   |        N/A        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
+
+" \*** " It is anticipated that this will be increased to Connext DDS 6.0.0 pending migration patches [7]_.
```

Comment:
I wouldn't say anticipated. Maybe rephrase as: "might be increased" if someone 
contributes pull requests for necessary changes".

Reply:
See ebec755